samples/sensor/hs300x/README.rst

```diff
@@ -0,0 +1,65 @@
+.. _hs300x_sample:
+
+HS300X Sample
```

Comment:
We want to stop introducing new driver-specific sensor samples, and instead use more generic ones for each sensor type. See `samples/sensor/accel_polling` and `samples/sensor/magn_polling` for example

Reply:
makes sense. do you want me to create a generic sample for temperature and humidity sensors, based on one of these samples? something like ```samples/sensor/temp_hum_polling``` perhaps?

Reply:
That would be great, thank you

Reply:
Maybe simply repurpose the `dht` sample?

Reply:
ok, I will look at creating a generic temperature/humidity sample based on the existing ```dht```sample.